### PR TITLE
Fix analysis tab when package version is outdated.

### PR DESCRIPTION
**Pull request by isoos**
When a package version is too old, we provide a link to the latest stable's analysis.

#888

### CODE REVIEW

File: app/lib/frontend/templates.dart [hunk 1]
```diff
@@ -191,6 +192,9 @@ class TemplateService {
         devDeps.isNotEmpty;
 
     final Map<String, dynamic> data = {
+      'package': package,
```
Comment (jakobr-google) [inline app/lib/frontend/templates.dart]: Tiny nit: We also use 'package' in the dependencies lists. I know it works due to scoping, but maybe use a different name here?

File: app/lib/frontend/templates.dart [hunk 2]
```diff
@@ -191,6 +192,9 @@ class TemplateService {
...
+      'show_outdated': analysis.analysisStatus == AnalysisStatus.outdated,
```
Comment (jakobr-google) [inline app/lib/frontend/templates.dart]: Nit: Could also use `{{^show_content}}` in the template to avoid passing in two booleans for this state.

Comment (isoos) [reply app/lib/frontend/templates.dart]: Updated, although with the other flag (using `{{^show_outdated}}` in the template)

File: app/lib/frontend/templates.dart (hunk 1 above)
Comment (isoos) [reply app/lib/frontend/templates.dart]: I kind of like the compact name and rely on the scope, but renaming can also work. Any suggestion on how they should be different?

Comment (jakobr-google) [reply app/lib/frontend/templates.dart]: It's fine as is. I just stumbled across it, and had to double-check how mustache worked.

**Pull request merged**